src/Form/DropdownField.jsx

```diff
@@ -84,6 +110,11 @@ export const DropdownField = ({
   )
 }
 
+const renderValueForMultipleOptions = options =>
+  options.length <= 2
+    ? options.slice(0, 2).join(', ')
+    : `${options.slice(0, 2).join(', ')} + ${options.length - 2} more`
```

Comment:
Need a `i18n` in props:

```
<Dropdown ...
   displayMoreThreshold={2}
  displayMorePattern={translate('dd_more')}
/>
```

where `dd_more` is something like `{0} und {1} mehr` for DE si `{0} and other` for EN.